sdcm/sct_events/setup.py

```diff
@@ -59,38 +56,6 @@ def start_events_device(log_dir: Optional[Union[str, Path]] = None,
     time.sleep(EVENTS_SUBSCRIBERS_START_DELAY)
 
     # Default filters.
```

Comment:
please add some reference here that we also do filtering in syslog-ng clients

Reply:
I left some of the filter in place, since they are not db log or k8s specific (which works without syslog-ng, at least now)